Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "buffrs"
-version = "0.8.1"
+version = "0.9.0"
```

Comment:
IMO the version bump should be done in another mr to separate concerns

Reply:
I kind of agree, although this change _is directly tied_ to a manifest change specifically and it should not be rolled out without the version bump